app/models/restaurant_reviews_metadatum.rb

```diff
@@ -0,0 +1,3 @@
+class RestaurantReviewsMetadatum < ApplicationRecord
```

Comment:
Recommended gem for database annotations: https://github.com/ctran/annotate_models

So you can tell what fields exist in a model.